create_app.py

```diff
@@ -105,9 +105,12 @@ def after_request(response):
         return response
 
     # TODO: Replace with an internal pagination mechanism
-    def calculate_total_pages(items_count: int) -> int:
+    def calculate_total_pages(items_count: Optional[int]) -> int:
```

Comment:
Doing a python upgrade in a separate PR I expect? So weird seeing Optional in a new repo that I'm reviewing 🤣

Reply:
I feel ya 🤣 I've been trying to support as many versions as possible but I don't know how much sense that actually makes. Either way, considering Python's on 3.12 right now, I'm not sure it's entirely necessary to support 3.9... 3 versions seem like more than enough, right?

Either way definitely a separate one to deal with 😅